content/en/developers/marketplace/offering.md

```diff
@@ -0,0 +1,266 @@
+4. [Install and run the Datadog Development Toolkit command](#install-and-run-the-datadog-development-toolkit).
+5. Create an [information tile-only listing](#create-an-informational-tile-only-listing) or a [full data integration](#create-a-full-data-integration).
+6. [Populate the integration tile scaffolding](#populate-the-tile-scaffolding). 
+7. [Open a pull request](#open-a-pull-request).
+8. [Review feedback and request approval to merge the pull request and release the integration tile](#obtain-approval-to-merge). 
+9. [Coordinate go-to-market opportunities with Partner Marketing](#coordinate-gtm-opportunities).
+
+### Select an offering
+
+A standard Marketplace integration tile appears with the following format:
+
+{{< img src="developers/marketplace/marketplace-tile-example.png" alt="Example Marketplace tile" style="width:30%" >}}
+
+Choose from the following offering types to create an integration tile that represents your listing on the [Datadog Marketplace][1]:
+
+- A [Datadog Agent-based integration](#build-an-agent-based-integration)
+- A [REST API integration](#build-a-rest-api-integration)
+- A [Datadog App](#build-a-datadog-app)
+- [SaaS license or subscription](#build-an-informational-tile-only-listing)
+- [Professional services](#build-a-bi-directional-data-integration)
+
+### Build an Agent-based integration
+
+{{< tabs >}}
+{{% tab "Datadog Agent-Based Integration" %}}
+
+An [OpenMetrics check][1] is suitable for gathering telemetry data from existing applications that expose metrics using the Open Metrics standard.
+
+
+[1]: https://docs.datadoghq.com/developers/custom_checks/prometheus/
+{{% /tab %}}
+{{% tab "Python Check" %}}
+
+A [Python check][1] is suitable for monitoring services or products that do not expose metrics in a standard format. The Python check is used to collect telemetry data from various APIs or command line tools.
+
+
+[1]: https://docs.datadoghq.com/developers/integrations/new_check_howto/?tab=configurationtemplate#write-the-check
+{{% /tab %}}
+{{% tab "DogStatsD" %}}
+
+[DogStatsD][1] is suitable for applications that already emit telemetry using the StatsD protocol. 
+
+Datadog adds additional Datadog-specific extensions to the StatsD protocol, including the following:
+
+- Histogram Metric Type
+- Service Checks
+- Events
+- Tagging
+
+
+[1]: https://docs.datadoghq.com/developers/dogstatsd/
+{{% /tab %}}
+{{< /tabs >}}
+
+For more information about Datadog Agent-based integrations, see [Introduction to Agent-based Integrations][2].
+
+### Build a REST API integration
+
+Use an [API integration][3] to enrich and submit data from your backend, or pull data directly out of Datadog. API integrations work well in building a connector between Datadog and another SaaS platform.
+ 
+Since API integrations do not use the Datadog Agent to collect data, you need to create an [informational tile-only listing](#) once your development work is complete.
```

Comment:
Good catch, thanks! 
```suggestion
Since API integrations do not use the Datadog Agent to collect data, you need to create an [informational tile-only listing](#build-an-informational-tile-only-listing) once your development work is complete.
```